packages/repository/test/acceptance/has-many.relation.acceptance.ts

```diff
@@ -191,4 +191,17 @@ describe('HasMany relation', () => {
   async function givenPersistedCustomerInstance() {
     return customerRepo.create({name: 'a customer'});
   }
+
+    async function givenPersistedCustomerInstanceWithOrder() {
```

Comment:
This is not used AFAICT, but do you have a test you'll add for it? I.e. from your comment:
>1 new test was added -Negative test for property.array POST reject.

Reply:
Hi @b-admike ,
thanks for your help.
i opened a new PR included the fix PR #2178